cli/cmd/configure_bridge.go

```diff
@@ -35,145 +24,24 @@ var bridgeCmd = &cobra.Command{
-		}
-		creds, err := retrieveBridgeCredentials(endpoint, apiToken)
-		if err != nil {
-			return err
-		}
-
-		fmt.Println("\nThese are your credentials")
-		fmt.Println("user: " + creds.User)
-		fmt.Println("password: " + creds.Password)
-		return nil
-	}
-	return configureBridgeCredentials(endpoint, apiToken, configureBridgeParams)
-}
-
-func getBridgeURLFromAPIURL(endpoint string) string {
-	parsedURL, err := url.Parse(endpoint)
-	if err != nil {
-		return ""
-	}
-	return strings.Replace(parsedURL.String(), parsedURL.Path, "/bridge", 1)
-}
-
-func retrieveBridgeCredentials(endpoint string, apiToken string) (*configureBridgeAPIPayload, error) {
-	client := &http.Client{
-		Transport: &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: true}, //nolint:gosec
-		},
-	}
-	req, err := http.NewRequest("GET", endpoint, nil)
-	if err != nil {
-		return nil, err
-	}
-	req.Header.Add("x-token", apiToken)
-	req.Header.Add("content-type", "application/json")
-
-	resp, err := client.Do(req)
-	if err != nil {
-		fmt.Println("Could not send request: " + err.Error())
-		return nil, err
-	}
-	defer resp.Body.Close()
-	body, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		return nil, err
-	}
-
-	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
-		return nil, errors.New("Received not successful response: " + string(body))
-	}
-
-	res := &configureBridgeAPIPayload{}
-	if err := json.Unmarshal(body, res); err != nil {
-		return nil, err
-	}
-	return res, nil
-}
-
-func configureBridgeCredentials(endpoint string, apiToken string, configureBridgeParams *configureBridgeCmdParams) error {
-	bridgeCredentials := configureBridgeAPIPayload{
-		User:     *configureBridgeParams.User,
-		Password: *configureBridgeParams.Password,
-	}
-
-	payload, err := json.Marshal(bridgeCredentials)
-	if err != nil {
-		fmt.Println("Could not marshal response payload: " + err.Error())
-		return err
-	}
-	client := &http.Client{
-		Transport: &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: true}, //nolint:gosec
-		},
-	}
-
-	req, err := http.NewRequest("POST", endpoint, bytes.NewReader(payload))
-	if err != nil {
-		return err
-	}
-	req.Header.Add("x-token", apiToken)
-	req.Header.Add("content-type", "application/json")
-
-	resp, err := client.Do(req)
-	if err != nil {
-		fmt.Println("Could not send request: " + err.Error())
-		return err
-	}
-	defer resp.Body.Close()
-	body, err := ioutil.ReadAll(resp.Body)
-
-	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
-		return errors.New("Received not successful response: " + string(body))
-	}
-
-	if err != nil {
-		return errors.New("Could not configure bridge: " + err.Error())
-	}
-	fmt.Println("Bridge credentials configured successfully")
-	return nil
-}
-
-func verifyConfigureBridgeParams(configureBridgeParams *configureBridgeCmdParams) error {
-	if configureBridgeParams.Read != nil && *configureBridgeParams.Read {
+		fmt.Println("This command is not supported anymore!")
```

Comment:
I think it would make sense to add the version that deprecated this command here

Reply:
done